fe/fe-core/src/main/java/com/starrocks/common/util/CompressionUtils.java

```diff
@@ -25,4 +25,22 @@ public class CompressionUtils {
     public static TCompressionType findTCompressionByName(String name) {
         return tCompressionByName.get(name);
     }
+
+    // Return TCompressionType according to input name for data page compression.
+    // Return null if input name is an invalid compression type for data page compression.
+    public static TCompressionType findPageTCompressionByName(String name) {
+        TCompressionType compressionType = tCompressionByName.get(name);
+
+        // storage support lz4, zlib, zstd for data page compression by now.
+        if (compressionType == null) {
+            return null;
+        } else if (compressionType == TCompressionType.LZ4
+                   || compressionType == TCompressionType.LZ4_FRAME
+                   || compressionType == TCompressionType.ZLIB
+                   || compressionType == TCompressionType.ZSTD) {
+            return compressionType;
+        } else {
```

Comment:
If the compression type is not supported, we should return an exception or error to give an obvious hint

Reply:
Yes, we do. The exception or error will be returned in PropertyAnalyzer which call this function.